cypress/support/pages/signIn.pageObject.js

```diff
@@ -1,18 +1,27 @@
 import PageObject from '../PageObject';
 
-class SignInPageObject extends PageObject {
-  url = '/#/login';
+class SignUpPageObject extends PageObject {
```

Comment:
The class name `SignUpPageObject` indicates this is a page object for sign-up, not sign-in. Ensure the file name matches the class name or intended functionality.